### PR TITLE
Josue ops course notes

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
Partially Completes [Ops Course AWS Lesson Documentation](https://app.shortcut.com/clarkcan/story/6921/ops-course-aws-lesson-documentation)

### CODE REVIEW

File: Ops/Ops-lessons-2022/AWS.md [hunk 1]
```diff
@@ -39,6 +95,14 @@
 
 ### Simple Storage Service (S3)
 
+In short S3 allows customers to store objects into buckets located in a region but are accessible by 'all' regions <br>
+You can find S3 documentation [here](https://docs.aws.amazon.com/s3/?id=docs_gateway)
+
+An object is a file and any metadata that describes the file. While a bucket is a container for objects. Each object has a key, which is the unique identifier for the object within the bucket <br>
+We use S3 buckets to store static websites, files, and logs.
+
+S3 buckets and the objects inside them are by default, private. You have access only to the S3 resources you created. Access can be granted by IAM permissions.
```
Comment (EVogel1999) [inline Ops/Ops-lessons-2022/AWS.md]: I would just add that S3 is also called blob storage.  Blobs (or objects) are files.

**Review by EVogel1999 — APPROVED** [review]
one small suggestion for s3 to include a definition for what are blobs and blob storage

**Review by mfranz1 — APPROVED** [review]

File: Ops/Ops-lessons-2022/AWS.md [hunk 2]
```diff
@@ -4,19 +4,75 @@
...
+
+AWS has regions all across the world and within each region there are availability zones which are one or more discrete data centers.
 
 ## AWS Interfaces
 
+There are 3 main interfaces used to communicate with AWS
+
+* AWS Console
+* SDKs (Software Development Kits)
+* AWS CLIv2
```
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md]:
```suggestion
* [AWS CLIv2](https://docs.aws.amazon.com/cli/latest/reference/)
```

File: Ops/Ops-lessons-2022/AWS.md [hunk 3]
```diff
@@ -4,19 +4,75 @@
...
 ### Identity and Access Management (IAM)
 
+In short [IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) is an access control service that provides ways to organize and manage privileges. <br>
+
+IAM allows you to give different permissions to all your employees in your organizations or anyone outside of your organization. 
+The biggest features are User groups, Roles, and Policies
+
+#### User Groups
+
+User Groups can be given to Users in an organizations. User Groups give users certain permissions to certain services. In our recent IAM refactor we created multiple user groups. 3 groups for every category (App Integration, Compute, Management, Networking, Security, and Storage) <br>
```
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md]:
```suggestion
User Groups are used to groups users in organizations. User Groups have policies attached to them that allow or deny permission to AWS services. In our recent IAM refactor we created multiple user groups. 3 groups for every category (App Integration, Compute, Management, Networking, Security, and Storage) <br>
```

File: Ops/Ops-lessons-2022/AWS.md [hunk 4]
```diff
@@ -4,19 +4,75 @@
...
+For example: Compute_1 gives a user full access to AmazonEC2, Administrator access to ElasticBeanstalk, and full access to AWSLambda, while Compute_3 only gives a user read access to these services. For a full breakdown to these user groups, please refer to the google drive document [here](https://docs.google.com/spreadsheets/d/1HKrcq5dDfvAB6Mqk9Ipa6GlcegCP4dWbXRYpjqgsbK8/edit?usp=sharing)
+
+#### Roles
+
+Roles are an identity you can crate that has specific permissions with credentials that are valid for a short duration.
```
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md]:
```suggestion
Roles are identities with attached policies that leverage dynamic credentials by making requests to STS.
```

File: Ops/Ops-lessons-2022/AWS.md [hunk 5]
```diff
@@ -4,19 +4,75 @@
...
+
+#### Policies
+
+Policies are attached to users, roles, and user groups. <br>
+There are AWS managed polices such as: AmazonEC2FullAccess. Users are also allowed to create their own policies to manage users and user groups. <br>
```
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md]:
```suggestion
AWS supports both AWS managed and customer managed policies. An example of AWS managed is AmazonEC2FullAccess 
</br>
```

File: Ops/Ops-lessons-2022/AWS.md (hunk 1 above)
Comment (Cwagne17) [inline Ops/Ops-lessons-2022/AWS.md]:
```suggestion
S3 buckets and the objects inside them are by default, private. You have access only to the S3 resources you created. Access can be granted through bucket policies, bucket ACLs, and IAM policies.
```

**Review by Cwagne17 — COMMENTED** [review]
Just a few additions, looks good.

**Review by StarChaser44 — APPROVED** [review]

**Review by jouypo23 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**